web/vibe/web/common.d

```diff
@@ -440,16 +442,23 @@ unittest {
 
 /**
  	Respresents a Rest error response
-*/
-class RestException : HTTPStatusException {
+*/class RestException : HTTPStatusException {
 	private {
 		Json m_jsonResult;
 	}
 
 	@safe:
 
+    ///
+	this (int status, string result, string file = __FILE__, int line = __LINE__, Throwable next = null)
```

Comment:
Any attributes that can go here, such as`@safe` and `nothrow` ?

Reply:
@safe already applied (@safe:) and other cannot go, but I will change the attributes on jsonResult in a separate commit